.gitignor

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
- [ ] GOOD JOB SO FAR 👏. You have successfully added your ` .gitignore` file but it is not working as expected. The reason why it's not working is that you have called it `.gitignor`. Please kindly fix this issue with adding e to end of your file name like this `.gitignore`